index.html

```diff
@@ -7,10 +7,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio: Ahmed Hasan Rony </title>
     <link rel="stylesheet" href="css/style.css">
+    <!-- fontawesome cdn -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 
-<body>
-    <h1 class="hello">Portfolio</h1>
+<body class="portfolio">
+    <header class="header">
+        <nav class="portfolio__navbar">
+            <div class="navbar__logo">
+                <a href=#>Welcome</a>
+            </div>
+            <div class="navbar-button">
+                <a href="#"><img src="images/button.png" alt="nav-button" height="20px"></a>
+            </div>
+        </nav>
+    </header>
+    <div class="main__content">
```

Comment:
- [ ] Since this is another section entirely, it should use the section tag. It represents a generic standalone section of the document, so I will advise you use it.

Reply:
ok I will use section tag